lib/takeaway.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'menu'
+require_relative 'order'
+require_relative 'text'
+
+class TakeAway
+  attr_reader :menu, :order, :text
+
+  def initialize (menu = Menu.new, order = Order.new(menu), text = Text.new)
+    @menu = menu
+    @order = order
+    @text = text
+  end
+
```

Comment:
I love the delegation in view_menu, `add` and `review`.  There is no logic in them and the message is completely delegated to the other classes.